htdocs/compta/bank/card.php

```diff
@@ -730,6 +736,14 @@
 				}
 			}
 			print '</td></tr>';
+			
+			print '<tr><td>'.$langs->trans("ICS").'</td>';
```

Comment:
If the module direct debit and credit transfer are not enabled, i think we can hide this information (most users don't know what it is related too when modules are not enabled). Donc(t you think so ?

Reply:
Yes, I agree. I will commit.

(Happy New Year 2021!!)